plugins/faustwp/phpunit.xml.dist

```diff
@@ -10,6 +10,9 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
 >
+  <php>
+    <env name="LOAD_PATCHWORK" value="1"/>
+  </php>
```

Comment:
The included unit tests [required patchwork](https://github.com/wpengine/faustjs/pull/1656/files#diff-7579f98151f8c0809460839a05615f888c9e7688d2c7f27dbc033b4339b4e330R7) to be loaded before the tests. Requiring patchwork in `plugins/faustwp/tests/bootstrap.php` solved the issue for the new tests but broke some of the Codeception E2E tests.

Codeception is a superset of PHPUnit, however it does not rely on the phpunit configs in this project
- `plugins/faustwp/phpunit.xml.dist`
- `plugins/faustwp/phpunit-multisite.xml.dist`

This allows us to define a custom environment variable, only including patchwork for the relevant **unit** and **integration** tests.